core/src/main/scala/com/salesforce/op/stages/impl/feature/OpScalarStandardScaler.scala

```diff
@@ -62,13 +62,13 @@ class OpScalarStandardScaler
 
     val std = scalerModel.std.toArray
     val mean = scalerModel.mean.toArray
-    if (std.length == 1) {
-      val stdMean = std.head
-      val meanMean = mean.head
-      val scalingArgs = LinearScalerArgs(1 / stdMean, - meanMean / stdMean)
-      val meta = ScalerMetadata(ScalingType.Linear, scalingArgs).toMetadata()
-      setMetadata(meta)
-    }
+
+    // Since is a UnaryEstimator, exactly one value will be in std and mean
```

Comment:
It's because it's a UnaryEstimator from `RealNN` to `RealNN`, if it were a UnaryEstimator with different type paramaters (eg. `OpVector` to `OpVector`) then it could have a full array of values in it/

Reply:
Added clarification in new commit